### DISCUSSION TITLE
Quartz.Net v4?

### DISCUSSION THREAD

**Discussion by drieseng** [quartznet/quartznet#1767] [General]
@lahma Will Quartz.Net v4 eventually contain all changes from the v4 branch? The **v4** branch is currently very much out of sync with **main**.

**Comment by lahma** [comment]
I think there shouldn't be anything there that hasn't been already taken into main via [this large merge](https://github.com/quartznet/quartznet/pull/1568). `main` is v4 and `3.x` branch is maintenance line.

**Reply by drieseng** [reply]
I based my comment on [this](https://github.com/quartznet/quartznet/compare/main...v4), but let me check if we missed something.
